livebooks/telemetry.livemd

```diff
@@ -0,0 +1,282 @@
+# Telemetry
```

Comment:
Wow. We can now run grpc server inside Livebook? *o*

Reply:
Yup! You need to have the rights to bind the port you'll run the server

Reply:
:)

I believe you can in theory run anything inside livebook if you try hard enough haha

In this case it was luckily not hard at all, and so the livebook both shows the way to use telemetry and works as a manual validation file for developing this PR